projects/demo-playwright/tests/kit/carousel/carousel.spec.ts

```diff
@@ -0,0 +1,38 @@
+import {tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`Carousel`, () => {
```

Comment:
Should we decrease viewport for this test ? 🤔 
![carousel-draggable-chromium-darwin](https://github.com/taiga-family/taiga-ui/assets/35179038/ff4a2840-a6ff-472b-baa9-601a501170af)

Reply:
Makes sense. Made it 360x500, I think the screenshots are not too wide now.